src/huggingface_hub/keras_mixin.py

```diff
@@ -97,6 +99,9 @@ def push_to_hub_keras(
             Specify this in case you want to push to an existing repository in the hub. If unspecified, a new
             repository will be created in your namespace (unless you specify an :obj:`organization`) with
             :obj:`repo_name`.
+        tensorboard_dir (:obj:`str`, `optional`):
+            TensorBoard logging directory. Specify this in case you have used TensorBoard callback and would like
+            to host TensorBoard on your model repository.
```

Comment:
WDYT of something along these lines?
```suggestion
            TensorBoard logging directory to be pushed. The Hub automatically hosts 
            and displays a TensorBoard instance if log files are included in the repository.
```